components/form/createProfileMain.tsx

```diff
@@ -12,84 +12,81 @@ import * as React from 'react';
-        <Paper variant="outlined" sx={{ my: { xs: 3, md: 6 }, p: { xs: 2, md: 3 } }}>
-          <Typography component="h1" variant="h4" align="center">
-            Create profile
-          </Typography>
-          <Stepper activeStep={activeStep} sx={{ pt: 3, pb: 5 }}>
-            {steps.map((label) => (
-              <Step key={label}>
-                <StepLabel>{label}</StepLabel>
-              </Step>
-            ))}
-          </Stepper>
-          <React.Fragment>
-            {activeStep === steps.length ? (
-              <React.Fragment>
-                <Typography variant="h5" gutterBottom>
-                  Thank you for registering with us.
-                </Typography>
-                <Typography variant="subtitle1">
-                  To cook your profile we have sent you an email for
-                  confirmation, and your profile will be served as soon as the verification is complete.
-                </Typography>
-              </React.Fragment>
-            ) : (
-                <React.Fragment>
-                  {getStepContent(activeStep)}
-                  <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
-                    {activeStep !== 0 && (
-                      <Button onClick={handleBack} sx={{ mt: 3, ml: 1 }}>
-                        Back
-                      </Button>
-                    )}
-                    <Button
-                      variant="contained"
-                      onClick={handleNext}
-                      sx={{ mt: 3, ml: 1 }}
-                    >
-                      {activeStep === steps.length - 1 ? 'Confirm' : 'Next'}
-                    </Button>
-                  </Box>
-                </React.Fragment>
-              )}
-          </React.Fragment>
-        </Paper>
-      </Container>
-    </ThemeProvider>
-  );
-}
\ No newline at end of file
+    return (
+        <ThemeProvider theme={theme}>
+            <Container component='main' maxWidth='sm' sx={{ mb: 4 }}>
+                <Paper variant='outlined' sx={{ my: { xs: 3, md: 6 }, p: { xs: 2, md: 3 } }}>
+                    <Typography component='h1' variant='h4' align='center'>
+                        Create profile
+                    </Typography>
+                    <Stepper activeStep={activeStep} sx={{ pt: 3, pb: 5 }}>
+                        {steps.map((label) => (
+                            <Step key={label}>
+                                <StepLabel>{label}</StepLabel>
+                            </Step>
+                        ))}
+                    </Stepper>
+                    <React.Fragment>
+                        {activeStep === steps.length ? (
+                            <React.Fragment>
+                                <Typography variant='h5' gutterBottom>
+                                    Thank you for registering with us.
+                                </Typography>
+                                <Typography variant='subtitle1'>
```

Comment:
@JRakin instead of writing tertiary condition inside markup better to make separate components